frontend/src/components/layouts/Header.js

```diff
@@ -22,12 +22,12 @@ export const Header = () => {
 					</Link>
 					<Link to="/communityConfiguration">
 						<li className="nav-item">
-							<a className="nav-link" >Package Generation</a>
+							<a className="nav-link" >Community Configuration</a>
```

Comment:
Plural form would be better. We are presenting more than one to the user.

Reply:
Done